h/realtime.py

```diff
@@ -58,22 +55,6 @@ def _random_id(self):
         data = struct.pack("Q", random.getrandbits(64))
         return base64.urlsafe_b64encode(data).strip(b"=")
 
-    def _record_time_in_queue(self, message):
-        """Send a very rough estimate of time-in-queue to the stats client."""
-        # N.B. This gives only a *rough approximation* of the time spent in the
-        # message queue. Using wall clocks like this is NOT going to be very
-        # accurate.
-        if "timestamp" not in message.headers:
-            return
-
-        timestamp = datetime.strptime(
-            message.headers["timestamp"], "%Y-%m-%dT%H:%M:%S.%fZ"
-        )
-        delta = datetime.utcnow() - timestamp
-        delta_millis = int(delta.total_seconds() * 1000)
-
-        self.statsd_client.timing("streamer.msg.queueing", delta_millis)
```

Comment:
Once upon a time we wanted to know how long things are spending in RabbitMQ: https://github.com/hypothesis/h/pull/3486/ We haven't used this for years